.gitignore

```diff
@@ -1,4 +1,5 @@
 bin
 coverage.txt
 profile.out
-coverage.out
\ No newline at end of file
+coverage.out
+.idea/
```

Comment:
You should ignore .idea on your general gitignore file

Reply:
done